{{cookiecutter.project_folder}}/pyproject.toml

```diff
@@ -0,0 +1,82 @@
+[tool.poetry]
+name =  "{{ cookiecutter.namespace + '-' + cookiecutter.module_name if cookiecutter.namespace else cookiecutter.module_name }}"
+version = "{{ cookiecutter.version }}"
+description = "{{ cookiecutter.short_description }}"
+authors = ["{{ cookiecutter.full_name }} <{{ cookiecutter.email }}>"]
+readme = "README.md"
+packages = [{include =  "{{ cookiecutter.namespace + '/' + cookiecutter.module_name if cookiecutter.namespace else cookiecutter.module_name }}"}]
+
+[tool.poetry.dependencies]
+python = ">={{ cookiecutter.python_version }}, <3.12"
+{%- if cookiecutter.use_jupyterlab == "y" %}
+jupyterlab = "^3.0.0"
+{%- endif %}
+{%- if "{{ cookiecutter.command_line_interface|lower }}" == "cli" %}
+typer-cli = "^0.0.13"
+{%- endif %}
+{%- if cookiecutter.use_mypy == "y" %}
+pydantic = "^1.10.7"
+{%- endif %}
```

Comment:
Can we add FastAPI and Pydantic as defaults here?

Reply:
I have added both of them without any conditionals. I added them not as dev dependencies though. Hopefully thats what you meant